arangod/Aql/ModificationExecutorInfos.cpp

```diff
@@ -52,6 +52,7 @@ ModificationExecutorInfos::ModificationExecutorInfos(
       _doCount(doCount),
       _isReplace(isReplace),
       _ignoreDocumentNotFound(ignoreDocumentNotFound),
+      _useBatching(true),  // used only by upsert
```

Comment:
Why is this necessary, do the other modification executors not batch as well?

Reply:
True, all other modification executors use batching.

The actual batch size however is only set in the ctors of `SimpleModifier` and `UpsertModifier`.
Probably we should remove the extra `_useBatching` flag and instead put the batch size into the `ModificationExecutorInfos`?

Reply:
Changed.

Reply:
If I recall correctly, the only modifier that is *not* simple is the upsert modifier, and previouslyl the upsert modifier was forced to use a batchsize of 1; so it should be correct to have the batch size in the ModificationExecutorInfos.

Reply:
I think so too. The change is implemented in the latest version of this PR.